packages/design-system/src/components/Badge/primitive/BadgePrimitive.tsx

```diff
@@ -82,7 +86,11 @@ function BadgePrimitive({
 				align="center"
 				display="inline"
 			>
-				<span className={classnames(styles.badge__label)} data-testid="badge-label">
+				<span
+					className={classnames(styles.badge__label)}
+					data-testid={dataTestId ? `${dataTestId}.${defaultTestId}` : defaultTestId}
```

Comment:
I would prefer to put it in a const and reuse it

Reply:
now test and testid are different